x/concentrated-liquidity/fees.go

```diff
@@ -121,6 +121,7 @@ func (k Keeper) initOrUpdateFeeAccumulatorPosition(ctx sdk.Context, poolId uint6
 }
 
 // getFeeGrowthOutside returns the sum of fee growth above the upper tick and fee growth below the lower tick
+// WARNING: this method may mutate the pool, make sure to refetch the pool after calling this method.
```

Comment:
Can you be specific about what in this method mutates the pool? Its not super clear to me here.

Reply:
added comment